masterlist.yaml

```diff
@@ -976,6 +1017,21 @@ plugins:
     url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/5941/' ]
     after:
       - 'Relationship Dialogue Overhaul.esp'
+  - name: 'XPMSE.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/1988/'
+        name: 'XP32 Maximum Skeleton Special Extended - XPMSSE on Nexus Mods'
+    msg:
+      - <<: *doNotClean
+      - <<: *requireSKYUI
+        subs: [ '[XP32 Maximum Skeleton Special Extended](https://www.nexusmods.com/skyrimspecialedition/mods/1988/)' ]
+        condition: 'file("scripts/XPMSEMCM.pex") and not active("SkyUI_SE.esp")'
+      - type: error
+        content: 'You have installed [XP32 Maximum Skeleton Special Extended](https://www.nexusmods.com/skyrimspecialedition/mods/1988/) features which require [RaceMenu Special Edition](https://www.nexusmods.com/skyrimspecialedition/mods/19080/) to function correctly, but it is not installed.'
+        condition: 'file("scripts/RaceMenuPluginXPMSE.pex") and not active("RaceMenu.esp")'
```

Comment:
Same as above, it was the easiest and most reliable way I could think of at the time.